### ISSUE TITLE
Build in scoped styles for components

### ISSUE THREAD

**Issue by knownasilya** [AppGeo/emberate#14]
I can see https://github.com/css-modules/css-modules being used for:

```no-highlight
components/
  my-component/
    component.js
    template.hbs
    style.css
```

And then you could inject that class name into the component object in the built index.js.

**Comment by CNDW** [comment]
is this being used by ember's built-in resolver?

**Comment by knownasilya** [comment]
No, Ember doesn't currently handle styles. Pods 2 will have it though.

**Comment by knownasilya** [comment]
https://github.com/salsify/ember-css-modules
